tests/cypress/e2e/features/requests_page.js

```diff
@@ -0,0 +1,302 @@
+    ) {
+        cy.contains('.cvat-header-button', 'Requests').click();
+        cy.contains('.cvat-requests-card', requestAction)
+            .within(() => {
+                innerCheck();
+                if (checkResourceLink) {
+                    cy.get('.cvat-requests-name').click();
+                }
+            });
+
+        if (checkResourceLink) {
+            cy.get('.cvat-spinner').should('not.exist');
+
+            if (resourceLink) {
+                cy.url().should('include', resourceLink);
+            } else {
+                cy.url().should('include', '/requests');
+            }
+        }
+    }
+
+    before(() => {
+        cy.visit('/auth/login');
+        cy.login();
+
+        cy.imageGenerator(imagesFolder, imageFileName, width, height, color, posX, posY, mainLabelName, imagesCount);
+        cy.createZipArchive(directoryToArchive, badAnnotationsPath);
+        cy.createZipArchive(directoryToArchive, archivePath);
+        cy.createZipArchive(emptyDirectoryToArchive, badArchivePath);
+
+        data.cloudStorageID = cy.attachS3Bucket(cloudStorageData);
+
+        const project = {
+            name: projectName,
+            label: mainLabelName,
+            attrName,
+            attrVaue: 'Requests attr',
+            multiAttrParams: false,
+            advancedConfiguration: false,
+        };
+        cy.goToProjectsList();
+        cy.createProjects(
+            project.name,
+            project.label,
+            project.attrName,
+            project.attrVaue,
+            project.multiAttrParams,
+            project.advancedConfiguration,
+        );
+        cy.openProject(project.name);
+        cy.url().then((url) => {
+            data.projectID = Number(url.split('/').slice(-1)[0].split('?')[0]);
+
+            const task = {
+                name: taskName,
+                label: mainLabelName,
+                attrName,
+                textDefaultValue: 'Requests attr',
+                dataArchiveName: `${imageFileName}.zip`,
+                multiAttrParams: false,
+                forProject: true,
+                attachToProject: true,
+                projectName,
+            };
+            cy.goToTaskList();
+            cy.createAnnotationTask(
+                task.name,
+                task.label,
+                task.attrName,
+                task.textDefaultValue,
+                archiveName,
+                task.multiAttrParams,
+                null,
+                task.forProject,
+                task.attachToProject,
+                task.projectName,
+            );
+
+            cy.goToTaskList();
+            cy.openTask(taskName);
+            cy.url().then((taskUrl) => {
+                data.taskID = Number(taskUrl.split('/').slice(-1)[0].split('?')[0]);
+                cy.getJobIDFromIdx(0).then((jobID) => {
+                    cy.headlessCreateObjects([rectanglePayload], jobID);
+                });
+            });
```

Comment:
headlessCreateProject -> returns project ID
headlessCreateTask -> returns task ID and job ID

Reply:
There is a problem #8297, which doesnt allow to test backups with file share created tasks/projects. Thats why we create them from local files